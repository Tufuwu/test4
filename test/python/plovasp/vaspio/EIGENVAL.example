    1    1    1    1
  0.1333597E+02  0.2587511E-09  0.2587511E-09  0.2587511E-09  0.5000000E-15
  1.000000000000000E-004
  CAR 
 V                                       
     11     10      9
 
  0.0000000E+00  0.0000000E+00  0.0000000E+00  0.8000000E-02
    1      -30.901243   1.000000
    2      -30.901242   1.000000
    3      -30.901242   1.000000
    4       -0.812822   1.000000
    5        6.116281   0.307472
    6        6.116282   0.000314
    7        6.116282   0.000000
    8        8.139559   0.000000
    9        8.139559   0.000000
 
  0.2000000E+00  0.0000000E+00  0.0000000E+00  0.9600000E-01
    1      -31.244548   1.000000
    2      -31.006017   1.000000
    3      -30.904003   1.000000
    4        0.680258   1.000000
    5        5.527848   0.997357
    6        5.837887   0.407717
    7        7.226963   0.000000
    8        7.781889   0.000000
    9        8.207361   0.000000
 
  0.4000000E+00  0.0000000E+00  0.0000000E+00  0.9600000E-01
    1      -31.771778   1.000000
    2      -31.171899   1.000000
    3      -30.908401   1.000000
    4        2.486834   1.000000
    5        4.687631   1.003771
    6        6.633241  -0.148217
    7        8.121131   0.000000
    8        8.444990   0.000000
    9        9.443636   0.000000
 
  0.2000000E+00  0.2000000E+00  0.0000000E+00  0.1920000E+00
    1      -31.593915   1.000000
    2      -31.156860   1.000000
    3      -30.976267   1.000000
    4        2.768444   1.000000
    5        4.858815   1.008186
    6        5.229541   1.085485
    7        7.709737   0.000000
    8        8.541449   0.000000
    9        9.490980   0.000000
 
  0.4000000E+00  0.2000000E+00 -0.5551115E-16  0.1920000E+00
    1      -31.739891   1.000000
    2      -31.337400   1.000000
    3      -30.997116   1.000000
    4        3.394894   1.000000
    5        4.629829   1.000179
    6        5.695587   0.810394
    7        8.098534   0.000000
    8        8.743598   0.000000
    9       12.244586   0.000000
 
  0.2000000E+00  0.2000000E+00  0.2000000E+00  0.6400000E-01
    1      -31.617656   1.000000
    2      -31.220150   1.000000
    3      -31.220150   1.000000
    4        4.450254   1.000000
    5        4.450254   1.000000
    6        4.634359   1.062133
    7        8.491526   0.000000
    8        8.491526   0.000000
    9       12.973660   0.000000
 
 -0.2000000E+00  0.2000000E+00  0.2000000E+00  0.4800000E-01
    1      -31.296977   1.000000
    2      -31.117861   1.000000
    3      -31.117861   1.000000
    4        1.927890   1.000000
    5        5.995969   0.879283
    6        6.272225  -0.010798
    7        6.272226   0.000000
    8        7.198547   0.000000
    9        8.910235   0.000000
 
 -0.4000000E+00  0.4000000E+00  0.2000000E+00  0.1920000E+00
    1      -31.682485   1.000000
    2      -31.294549   1.000000
    3      -31.142973   1.000000
    4        3.381464   1.000000
    5        4.651227   1.002211
    6        5.929509   0.183505
    7        7.231965   0.000000
    8        9.259558   0.000000
    9       10.660847   0.000000
 
 -0.4000000E+00 -0.4000000E+00  0.2000000E+00  0.6400000E-01
    1      -31.585360   1.000000
    2      -31.585359   1.000000
    3      -31.188820   1.000000
    4        3.846295   1.000000
    5        3.846295   1.000298
    6        5.810197   0.324984
    7        8.723375   0.000000
    8        8.723376   0.000000
    9       15.381038   0.000000
 
 -0.4000000E+00  0.4000000E+00  0.4000000E+00  0.4800000E-01
    1      -31.554619   1.000000
    2      -31.488158   1.000000
    3      -31.488158   1.000000
    4        3.090314   1.000000
    5        3.105025   1.000861
    6        8.500437  -0.068292
    7        8.500437   0.000000
    8        9.144518   0.000000
    9       13.491662   0.000000
