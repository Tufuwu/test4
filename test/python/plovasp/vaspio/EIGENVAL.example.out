nq = 1
ispin = 1
nelect = 11
nktot = 10
nband = 9
kpts:
[[ 0.   0.   0. ]
 [ 0.2  0.   0. ]
 [ 0.4  0.   0. ]
 [ 0.2  0.2  0. ]
 [ 0.4  0.2 -0. ]
 [ 0.2  0.2  0.2]
 [-0.2  0.2  0.2]
 [-0.4  0.4  0.2]
 [-0.4 -0.4  0.2]
 [-0.4  0.4  0.4]]
kwghts:
[ 0.008  0.096  0.096  0.192  0.192  0.064  0.048  0.192  0.064  0.048]
eigs:
[[[-30.901243]
  [-30.901242]
  [-30.901242]
  [ -0.812822]
  [  6.116281]
  [  6.116282]
  [  6.116282]
  [  8.139559]
  [  8.139559]]

 [[-31.244548]
  [-31.006017]
  [-30.904003]
  [  0.680258]
  [  5.527848]
  [  5.837887]
  [  7.226963]
  [  7.781889]
  [  8.207361]]

 [[-31.771778]
  [-31.171899]
  [-30.908401]
  [  2.486834]
  [  4.687631]
  [  6.633241]
  [  8.121131]
  [  8.44499 ]
  [  9.443636]]

 [[-31.593915]
  [-31.15686 ]
  [-30.976267]
  [  2.768444]
  [  4.858815]
  [  5.229541]
  [  7.709737]
  [  8.541449]
  [  9.49098 ]]

 [[-31.739891]
  [-31.3374  ]
  [-30.997116]
  [  3.394894]
  [  4.629829]
  [  5.695587]
  [  8.098534]
  [  8.743598]
  [ 12.244586]]

 [[-31.617656]
  [-31.22015 ]
  [-31.22015 ]
  [  4.450254]
  [  4.450254]
  [  4.634359]
  [  8.491526]
  [  8.491526]
  [ 12.97366 ]]

 [[-31.296977]
  [-31.117861]
  [-31.117861]
  [  1.92789 ]
  [  5.995969]
  [  6.272225]
  [  6.272226]
  [  7.198547]
  [  8.910235]]

 [[-31.682485]
  [-31.294549]
  [-31.142973]
  [  3.381464]
  [  4.651227]
  [  5.929509]
  [  7.231965]
  [  9.259558]
  [ 10.660847]]

 [[-31.58536 ]
  [-31.585359]
  [-31.18882 ]
  [  3.846295]
  [  3.846295]
  [  5.810197]
  [  8.723375]
  [  8.723376]
  [ 15.381038]]

 [[-31.554619]
  [-31.488158]
  [-31.488158]
  [  3.090314]
  [  3.105025]
  [  8.500437]
  [  8.500437]
  [  9.144518]
  [ 13.491662]]]
ferw:
[[[ 1.      ]
  [ 1.      ]
  [ 1.      ]
  [ 1.      ]
  [ 0.307472]
  [ 0.000314]
  [ 0.      ]
  [ 0.      ]
  [ 0.      ]]

 [[ 1.      ]
  [ 1.      ]
  [ 1.      ]
  [ 1.      ]
  [ 0.997357]
  [ 0.407717]
  [ 0.      ]
  [ 0.      ]
  [ 0.      ]]

 [[ 1.      ]
  [ 1.      ]
  [ 1.      ]
  [ 1.      ]
  [ 1.003771]
  [-0.148217]
  [ 0.      ]
  [ 0.      ]
  [ 0.      ]]

 [[ 1.      ]
  [ 1.      ]
  [ 1.      ]
  [ 1.      ]
  [ 1.008186]
  [ 1.085485]
  [ 0.      ]
  [ 0.      ]
  [ 0.      ]]

 [[ 1.      ]
  [ 1.      ]
  [ 1.      ]
  [ 1.      ]
  [ 1.000179]
  [ 0.810394]
  [ 0.      ]
  [ 0.      ]
  [ 0.      ]]

 [[ 1.      ]
  [ 1.      ]
  [ 1.      ]
  [ 1.      ]
  [ 1.      ]
  [ 1.062133]
  [ 0.      ]
  [ 0.      ]
  [ 0.      ]]

 [[ 1.      ]
  [ 1.      ]
  [ 1.      ]
  [ 1.      ]
  [ 0.879283]
  [-0.010798]
  [ 0.      ]
  [ 0.      ]
  [ 0.      ]]

 [[ 1.      ]
  [ 1.      ]
  [ 1.      ]
  [ 1.      ]
  [ 1.002211]
  [ 0.183505]
  [ 0.      ]
  [ 0.      ]
  [ 0.      ]]

 [[ 1.      ]
  [ 1.      ]
  [ 1.      ]
  [ 1.      ]
  [ 1.000298]
  [ 0.324984]
  [ 0.      ]
  [ 0.      ]
  [ 0.      ]]

 [[ 1.      ]
  [ 1.      ]
  [ 1.      ]
  [ 1.      ]
  [ 1.000861]
  [-0.068292]
  [ 0.      ]
  [ 0.      ]
  [ 0.      ]]]
